### ISSUE TITLE
The genes list

### ISSUE THREAD

**Issue by mcpecommander** [TheUnderTaker11/GeneticsReborn#12]
So I going to make a list including all of the genes and which are allowed to certain mobs.
This list is going to be long , so please do not close this issue until we agree on the whole list

**Comment by mcpecommander** [comment]
the first one is for the creeper: he should have this auto exploding gene, which should be only allowed to be given to passive mobs like chickens and cows , so that when you kill them, they explode instead. In my opinion it will be a lot of fun to be used in servers and defense mechanisms

**Comment by mcpecommander** [comment]
The second one is the skeleton which is a great suggestion: it should have the wolf magnet gene, I know it seems silly but the idea is to this gene allowed to be given to every mob except for flying ones (I assume it will break a few flying patterns) but the good part is when you make the anti plasmid , it will make the wolf repel gene which can be great in PVP as it will stop wolfs from attacking the one that has this gene whether it was a player or a mob, and it can be great to protect animal farms from stray wolfs

**Comment by TheUnderTaker11** [comment]
The anti plasmid has to be something that removes the gene completely. I can't break that concept for just a few genes. The explosion thing is funny enough that it's worth adding, and it will be easy to do since I can just cause an explosion at the XYZ of their death. For the player I'm going to make it useable, but only if there is gunpowder in their inventory.(If they could easily blow up their own home, who would use it?)

About the wolf thing, it's meh. First off, nobody in the modded MC scene every uses wolfs, for anything in any way. I would know I used to play lots of modded. Secondly out of the people that do enjoy playing with like 3 mods(HOW?) its very unlikely they have even a small server for that, and even LESS likely they will use wolfs as a big thing in PVP. 
To sum up the wolf thing, it would take a lot of work and pretty much no one would use it.

When you said the word "Magnet" it gave me an idea though, but I don't know what mob it could drop from besides enderman or endermite. The magnet gene, which makes all items in X range be "pulled" into your inventory, unless you are shifting, and I'll probably make a keybind to toggle it.

**Comment by mcpecommander** [comment]
That is great but for the magnet stuff make allowed for donkeys too, making them collect items around them

**Comment by TheUnderTaker11** [comment]
Added two separate genes for XP and Item magnet, also added a gene called "Explosive exit". Mobs will explode when they die but players only will if they have at least 5 gunpowder in their inventory. It won't use it, but you probably will lose everything from the explosion anyways ;P
Will be out whenever I release v0.19
